README.md

```diff
@@ -75,18 +75,23 @@ const PrivateEth = async () => {
   }`
 
   const config = {
-    "bootnodeEnodes": [ // --bootnodesv5 / Enodes of v5 bootnodes for p2p discovery
-      "enode://XXXX@X[::]:XXXX",
-      "enode://YYYY@Y[::]:YYYY"
+    bootnodeEnodes: [ // --bootnodesv5 / Enodes of v5 bootnodes for p2p discovery
+      'enode://XXXX@X[::]:XXXX',
+      'enode://YYYY@Y[::]:YYYY'
     ],
-    "networkID": networkID, // --networkid / Network identifier (integer, 0=Olympic (disused), 1=Frontier, 2=Morden (disused), 3=Ropsten) (default: 1)
-    "maxPeers": 0, // --maxpeers / Maximum number of network peers (network disabled if set to 0) (default: 25)
-    "genesis": genesis, // genesis.json file
-    "nodeDir": ".private-ethereum", // --datadir / Data directory for the databases and keystore
-    "keyStoreDir": "keystore", // --keystore / Directory for the keystore (default = inside the datadir)
-    "enodes": "enode://XXXX@X[::]:XXXX" // static_nodes.json file. Comma separated enode URLs
-    "noDiscovery": false, // --nodiscover / determines if the node will not participate in p2p discovery (v5)
-    "syncMode": 5 // the number associated with a sync mode in `celo-blockchain/mobile/geth.go`
+    networkID: networkID, // --networkid / Network identifier (integer, 0=Olympic (disused), 1=Frontier, 2=Morden (disused), 3=Ropsten) (default: 1)
+    maxPeers: 0, // --maxpeers / Maximum number of network peers (network disabled if set to 0) (default: 25)
+    genesis: genesis, // genesis.json file
+    nodeDir: '.private-ethereum', // --datadir / Data directory for the databases and keystore
+    keyStoreDir: 'keystore', // --keystore / Directory for the keystore (default = inside the datadir)
+    enodes: 'enode://XXXX@X[::]:XXXX', // static_nodes.json file. Comma separated enode URLs
+    noDiscovery: false, // --nodiscover / determines if the node will not participate in p2p discovery (v5)
+    syncMode: 5, // the number associated with a sync mode in `celo-blockchain/mobile/geth.go`
+    // HTTP RPC server - only intended for development & debugging
+    httpHost: '0.0.0.0', // host of the server
```

Comment:
I don't think we should suggest opening RPC on `0.0.0.0:8545` because that could allow anyone on the local network to access the node and completely compromise the node, possible also attacking the device through the javascript execution features allowed by the `debug` API. Opening on `127.0.0.1:8545` is better. With this, we should definitely add a stronger warning here about only using this in debug mode because opening the network socket allows any app on the device to access the node with full permissions.